src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn({
+                'is-active': selectedTab.id === tab.id,
+              })}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (selectedTabId !== tab.id) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
same. better to make function in component and pass referense on it here.
```
const handleClick = () => {
  if (selectedTabId !== tab.id) {
    onTabSelected(tab);
  }
}
```
```suggestion
                onClick={handleClick}
```

Reply:
It doesn't work properly(

Reply:
I agree with Vitaliy. Please make it. If you have questions, ask in the chat

You just need to move this logic to a function and use it here